CMakeLists.txt

```diff
@@ -205,6 +226,7 @@ set(_acts_boost_recommended_version 1.78.0)
 # required packages
 if (ACTS_SETUP_BOOST)
   if (ACTS_USE_SYSTEM_BOOST)
+    message(STATUS "Using system installation of Boost")
```

Comment:
This is just temporary, right?

CMake out of the box can indeed be short on info on what it is looking for, but there are better ways of printing more info in case the `find_package(Boost)` call doesn't provide enough of it. :thinking: 

(The same comment goes for all similar printouts...)

Reply:
Ok, let's move this to another PR as another cleanup.